tracer/test/test-applications/integrations/Samples.AzureServiceBus/Samples.AzureServiceBus.csproj

```diff
@@ -1,12 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">7.13.0</ApiVersion>
+
     <!-- System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.1-->
     <!-- https://andrewlock.net/stop-lying-about-netstandard-2-support/-->
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.14.0" />
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="$(ApiVersion)" />
```

Comment:
Just out interest, why choose 7.13.0 as the default when we had 7.14.0 before?

Reply:
No particular reason, though 7.13.0 was the first version to share the same snapshot results as 7.14.0 so I guess I just entered it in there 🤷🏼